ddp-workspace/projects/ddp-sdk/src/lib/components/user/userMenu.component.ts

```diff
@@ -51,7 +59,15 @@ export class UserMenuComponent {
         this.router.navigateByUrl('dashboard');
     }
 
+    public openPrism(): void {
+        this.router.navigateByUrl('prism');
```

Comment:
Other studies might have Prism at a different URL path. Anyway to get this from config?

Reply:
done